login.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="login.css">
+    <title>Blog</title>
+</head>
+<body>
+    <section>
+        <div class="welcome">
+            <h2 class="font">Welcome to</h2>
+            <h1 class="font" id="main-font">My Blog</h1>
+        </div> 
+        <div>
+            <form class="login-section">
+                <fieldset>

```

Comment:
What about adding a legend which shows Login?

Reply:
Great idea .. let me first practise using it😂